### PR TITLE
api updated

### PR DESCRIPTION
**Pull request by shivam-webkul**
**BUGS:**
 added the option for the all types products.

### CODE REVIEW

File: packages/Webkul/API/Http/Resources/Catalog/Product.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function toArray($request)
             'name'                   => $this->name,
             'url_key'                => $this->url_key,
             'price'                  => $product->getTypeInstance()->getMinimalPrice(),
-            'formated_price'         => core()->currency($product->getTypeInstance()->getMinimalPrice()),
+            'formatted_price'         => core()->currency($product->getTypeInstance()->getMinimalPrice()),
```
Comment (SteffenMahler) [inline packages/Webkul/API/Http/Resources/Catalog/Product.php:40]: Keep in mind that this is a breaking change.

Live systems, that are using this api endpoint, will still use 'formated_price' and will fail after this change.
There are also a lot of views, that aren't updated according to this renaming.

Best regards

Steffen

Comment (shivam-webkul) [reply packages/Webkul/API/Http/Resources/Catalog/Product.php:40]: Thanks @SteffenMahler ,

I will revert this changes.

Comment (SteffenMahler) [reply packages/Webkul/API/Http/Resources/Catalog/Product.php]: You are welcome.

**Comment by rahulcs0082** [comment]
Hi @shivam-webkul 

Kindly resolve conflict.

**Comment by shubhwebkul** [comment]
@devansh-webkul please request a pull with these changes

**Pull request closed without merging**